src/app/components/left-panel/left-panel.component.html

```diff
@@ -0,0 +1 @@
+<p>left-panel works!</p>
```

Comment:
Please after you create the nav bar start to work on this component is completely empty